Sources/Models/BundleIdCapabilitiesResponse.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  BundleIdCapabilitiesResponse.swift
+//  AppStoreConnect-Swift-SDK
+//
+//  Created by Patrick Balestra on 12/23/19.
+//
+
+import Foundation
+
+/// A response containing a list of related resource IDs.
+public struct BundleIdCapabilitiesResponse: Codable {
```

Comment:
Every other model conforms to `Codable`. Searching for types conforming to `Codable` results in 418 results, searching for `Decodable` yields only 15 results (mostly related to this PR actually). Should we for consistency keep everything conform to `Codable`? I change change all instances in this PR to conform to `Codable` instead of just `Decodable` like the rest of the codebase.

Reply:
Thanks for looking into this @BalestraPatrick! I'm happy to stick to the conventions already in use in the project.